### DISCUSSION TITLE
câu 33333333333

### DISCUSSION THREAD

**Discussion by NKLinhNgan-10A02-stt19** [phhnhan-10A02-stt21/lop-10A02-Nhom8#6] [Q&A]
Bài toán: vẽ hình tam giác vuông lên bảng.Thuật toán nào được xem là thuật
toán giải bài toán? Tại sao?

**Comment by phhnhan-10A02-stt21** [comment]
chắc là cái số 2

**Comment by maidangiu** [comment]
ă tr oi

**Comment by maidangiu** [comment]
ũa nhớm mình nhớm mấy zạ

**Comment by maidangiu** [comment]
quên mất tiu gòiiii
